src/ParallelAlgorithms/Events/ObserveTimeStep.hpp

```diff
@@ -50,6 +52,34 @@ using ObserveTimeStep =
     ::Registration::Registrar<Events::ObserveTimeStep, Metavariables>;
 }  // namespace Registrars
 
+namespace detail {
+  using ObserveTimeStepReductionData = Parallel::ReductionData<
```

Comment:
The formatting here is off. Would you please run clang-format over this file? (It will also automatically sort the includes alphabetically for you). You can try this: `git-clang-format upstream/develop`